src/style.css

```diff
@@ -1 +1,61 @@
 /* Styles go here */
+body {
+  margin: 0;
+  --color-blue: #00acdc;
+}
+
+.header {
+  display: flex;
+  padding: 0 50px;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  justify-content: space-between;
+  cursor: pointer;
```

Comment:
The 'cursor: pointer' property should be applied to .nav__link instead of .header, as it should change the cursor only when hovering over the links.